src/videotile/DefaultVideoElementResolutionMonitor.ts

```diff
@@ -32,12 +32,15 @@ export default class DefaultVideoElementResolutionMonitor implements VideoElemen
     this.observerQueue.delete(observer);
   }
 
-  bindVideoElement(newElement: HTMLVideoElement): void {
-    this.element = newElement;
-    if (!this.element) {
-      this.resizeObserver.unobserve(this.element);
+  bindVideoElement(newElement: HTMLVideoElement | null): void {
+    if (!newElement) {
+      if (this.element) {
+        this.resizeObserver.unobserve(this.element);
+      }
+      this.element = newElement;
```

Comment:
explicit `this.element = null` should be good.

Reply:
I like this change, makes it a bit easier to read

Reply:
I noticed `ResizeObserver` can observe multiple elements. So I updated the function to unobserve the element being observed if not `null`, and then observe `newElement` if not null. Also added a function to skip when `newElement` equals to the one being observed